schema/examples/direct/manifest.xml

```diff
@@ -2,11 +2,14 @@
 <direct-signature-job-manifest xmlns="http://signering.posten.no/schema/v1">
     <signer>
         <personal-identification-number>12345678910</personal-identification-number>
+        <signing-ceremony>
```

Comment:
@SindreNordbo @fredva Lenge siden vi spesifiserte dette også. Bør jo bare ta det for god fisk, og adlyde spesifikasjonen såklart, meeeeeeeen:

Noe i mot at det blir `<signing-method>AUTHENTICATED_SIGNATURE</signing-method>` og dropper hele `signing-ceremony`-elementet?